html/html.go

```diff
@@ -100,6 +101,15 @@ func (p *html) generateFormFunc(servName string, method *descriptor.MethodDescri
 	</div>`
 	p.P(`var Form`, servName, "_", generator.CamelCase(method.GetName()), " string = `", s, "`")
 }
+func processFileName (inStr string) string {
+	reg, err := regexp.Compile("[^a-zA-Z0-9]+")
+	if err != nil {
+		panic(err)
+	}
+	s := strings.TrimSuffix(inStr, ".proto")
+	processed := strings.Replace(strings.Title(reg.ReplaceAllString(s," ")), " ", "", -1)
+	return processed
+}
 
 func (p *html) Generate(file *generator.FileDescriptor) {
```

Comment:
Technically the html struct has state.
So it can remember to only generate some things once itself per package, without using a flag.
What do you think about that proposal?

Sorry for pushing this, but I want to try and keep things as simple as possible.

Reply:
Then we also don't need custom HeaderFooterName